Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM docker.io/centos:centos7
 
 RUN yum install -y python-requests && \
-    curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - current && \
+    curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - -b train current-tripleo && \
```

Comment:
This change is not from the commit

Reply:
this is to align the Dockerfile from metal3-io, although it's not necessary as it's not needed for this repo, Dockerfile.ocp is used instead
I can remove the change or just add a note in the commit